internal/super/flix.go

```diff
@@ -177,14 +203,20 @@ func packageCmd(
 	if err != nil {
 		return nil, err
 	}
-
 	isLocal := false
 	if getType(flixQuery) == flixPath {
 		isLocal = true
+		if flags.Save != "" {
+			// resolve template file location to relative path to be used by binding file
+			flixQuery, err = GetRelativePath(flixQuery, flags.Save)
```

Comment:
should be swapped also

Reply:
it's correct, I updated the method comment and params to be more clear.